packages/widget-chat/src/containers/chat-widget/index.js

```diff
@@ -357,17 +361,33 @@ export class ChatWidget extends Component {
 
       let scrollButton;
       if (props.widget.showScrollToBottomButton) {
-        const label = widget.hasNewMessage ? `New Messages` : null;
+        const newMessages = {
```

Comment:
I think i vaguely remember that language constants should be declared at the top of the file instead of the function that uses them. that could also be a personal style thing that i decided seemed like a rule 8 months ago.